docs/design/clustering/Makefile

```diff
@@ -14,3 +14,16 @@ $(FONT):
 
```

Comment:
This doesn't seem to have your last change where you fixed the curl target (removing the .ttf extension).